InstantSearch.js/getting-started/package.json

```diff
@@ -9,6 +9,7 @@
     "lint:fix": "npm run lint -- --fix"
   },
   "devDependencies": {
+    "@types/algoliasearch": "3.30.1",
```

Comment:
is this really necessary? VS Code should pick those up automatically 😄

Reply:
That was added by CodeSandbox automatically. We can remove it.